hpx/runtime/components/server/destroy_component.hpp

```diff
@@ -78,15 +78,6 @@ namespace hpx { namespace components { namespace server
 
         destroy<Component>(gid, addr, ec);
     }
-
-    ///////////////////////////////////////////////////////////////////////////
```

Comment:
Why hjave you removed these functions? Those are not specific to lcos::promise, are they?